cmd/cstor-pool-mgmt/controller/backup-controller/handler.go

```diff
@@ -217,6 +217,8 @@ func IsOnlyStatusChange(oldbkp, newbkp *apis.CStorBackup) bool {
 	return false
 }
 
+// updateCStorCompletedBackup updates the CStorCompletedBackups resource for the given backup
+// CStorCompletedBackups stores the information of last two completed backups
```

Comment:
As we are adding more comments, can you describe what information should be in: SnapName and PrevSnapName. 

For example, if snap1 was taken at t1, snap2 at t2 and t2 > t1, will snap1 be in SnapName or PrevSnapName?

Reply:
done